Add plugin: Wheel Tab Switcher

22-2 opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin:
https://github.com/22-2/wheel-tab-switcher

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [ ] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[if (window.Capacitor.getPlatform() !== "web") {](https://github.com/22-2/wheel-tab-switcher/blob/5da403687411f30ec408d5338e6fccbc4710f324/src/main.ts#L62-L64)
This code will never be called, because you have set `isDesktopOnly` to true.
Remove it.

[new Setting(this.containerEl).setName("⚙️ Wheel Tab Switcher Settings").setHeading();](https://github.com/22-2/wheel-tab-switcher/blob/5da403687411f30ec408d5338e6fccbc4710f324/src/settings.ts#L12)
[Don't add a top-level heading in the settings tab, such as "General", "Settings", or the name of your plugin.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)

[.setName("Top Bar Wheel Tab Switching (🚧 experimental)")](https://github.com/22-2/wheel-tab-switcher/blob/5da403687411f30ec408d5338e6fccbc4710f324/src/settings.ts#L15)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

22-2 commented:
@joethei Thank you for the review! I've addressed the feedback in this commit: https://github.com/22-2/wheel-tab-switcher/commit/64867b12b11d68d7830d610a8d9fd8095753f643

joethei commented:
[# Obsidian Wheel Tab Switcher](https://github.com/22-2/wheel-tab-switcher/blob/6dc72514c9a62930f0fe446da48cd6b6924c4d73/README.md?plain=1#L1C1)
Naming something "Obsidian xyz" is reserved for first party products we create, please change this header.
